src/Uno.Toolkit.UI/Controls/AutoLayout/AutoLayout.Layouting.Measure.cs

```diff
@@ -43,27 +43,28 @@ protected override Size MeasureOverride(Size availableSize)
 
 		// 2. Remove applicable padding and spacing from the remaining size
 		var paddingSize = Padding.GetLength(orientation);
+		var counterPaddingSize = Padding.GetCounterLength(orientation);
 		Decrement(ref remainingSize, paddingSize);
 
 		// 3. Measure fixed children
-		MeasureFixedChildren(orientation, availableCounterSize, ref remainingSize, ref desiredCounterSize);
+		MeasureFixedChildren(orientation, availableCounterSize, ref remainingSize, ref desiredCounterSize, counterPaddingSize);
 
 		// 4. Establish the total spacing size, if applicable
 		var totalSpacingSize = 0d;
 		if (numberOfStackedChildren > 0
-		    && justify != AutoLayoutJustify.SpaceBetween
-		    && spacing != 0
-		    && spacing < double.PositiveInfinity)
+			&& justify != AutoLayoutJustify.SpaceBetween
+			&& spacing != 0
+			&& spacing < double.PositiveInfinity)
 		{
 			totalSpacingSize = (numberOfStackedChildren - 1) * spacing;
 			Decrement(ref remainingSize, totalSpacingSize);
 		}
 
 		// 5. Calculate the size of Hug children
-		MeasureHugChildren(orientation, availableCounterSize, ref remainingSize, ref desiredCounterSize);
+		MeasureHugChildren(orientation, availableCounterSize, ref remainingSize, ref desiredCounterSize, counterPaddingSize);
 
 		// 6. Calculate the size of Filled children
-		MeasureFilledChildren(orientation, availableCounterSize, ref remainingSize, ref desiredCounterSize);
+		MeasureFilledChildren(orientation, availableCounterSize, counterPaddingSize, ref remainingSize, ref desiredCounterSize);
```

Comment:
Why the parameter ordering is different from those 2 methods? It's purely cosmetic, but that's nagging me ;-)

Reply:
fair enought I will fixe it ^^